src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -110,6 +110,12 @@
     <EnvironmentVariables Include="DeterministicSourcePaths=false" Condition="'$(DeterministicBuildOptOut)' == 'true'" />
 
     <EnvironmentVariables Include="SourceRoot=$(ProjectDirectory)" />
+
+    <!-- https://github.com/dotnet/source-build/issues/3081 -->
```

Comment:
@ellahathaway - looks like the internal flow is trying to add this back.  Can you update this PR accordingly?